packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -243,6 +251,14 @@ export interface VideoGalleryProps {
    * @defaultValue 'HorizontalBottom'
    */
   overflowGalleryPosition?: OverflowGalleryPosition;
+  /* @conditional-compile-remove(click-to-call) */
+  /**
+   * Determines the aspect ratio of local video tile in the video gallery.
+   * @remarks 'matchStream' will be responsive to the screen orientation and will change between 9:16 (portrait) and
```

Comment:
For matchstream, what is the aspect ratio when the camera is off?

Reply:
it will match the orientation like before. since we aren't getting a stream here it would be using the device information.